_search-plugins/knn/knn-index.md

```diff
@@ -48,14 +48,90 @@ However, if you intend to just use painless scripting or a k-NN score script, yo
  }
  ```
 
-## Method Definitions
+ ### Lucene byte vector
+
+By default, k-NN vectors are `float` vectors, where each dimension is 4 bytes. If you want to save storage space, you can use `byte` vectors with the `lucene` engine. In a `byte` vector, each dimension is a signed 8-bit integer in the [-128, 127] range. 
+ 
+Byte vectors are supported only for the `lucene` engine. They are not supported for the `nmslib` and `faiss` engines.
+{: .note}
+ 
+ To use a `byte` vector, set the `data_type` parameter to `byte` when creating mappings for an index:
```

Comment:
Shall we also add a point here saying that `data_type` is an optional parameter and it's default value is `float` ?

Reply:
Looks good. Thanks for making the changes.